src/BuildingBlocks/Service/Masa.BuildingBlocks.Service.Caller/Options/CallerOptions.cs

```diff
@@ -28,6 +28,8 @@ public Assembly[] Assemblies
 
     public string? RequestIdKey { get; set; }
 
```

Comment:
Line 8. Use an immutable collection or reduce the accessibility of the non-private readonly field 'Callers'.

Reply:
Line 19. Remove this argument from the method call; it hides the caller information.